support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -1,16 +1,17 @@
 package com.gu.emailservices
 
+import com.gu.i18n.Currency
 import com.gu.salesforce.Salesforce.SfContactId
+import com.gu.support.promotions.Promotion
+import com.gu.support.workers.{BillingPeriod, User}
 import io.circe.generic.auto._
 import io.circe.syntax._
 
-// scalastyle:off
```

Comment:
scalastyle doesn't seem to complain or edit this block when I compile, so I removed it